sysstat/map.jinja

```diff
@@ -1,9 +1,26 @@
-{% set sysstat = salt['grains.filter_by']({
-    'Debian': {
-        'pkg': 'sysstat',
-        'config': '/etc/default/sysstat',
-    },
-    'RedHat': {
-        'pkg': 'sysstat',
-    }
-}, merge=salt['pillar.get']('sysstat:lookup')) %}
+# -*- coding: utf-8 -*-
+# vim: ft=jinja
+
+{## Start with  defaults from defaults.sls ##}
+{% import_yaml 'sysstat/defaults.yml' as default_settings %}
+
+{## setup variable using grains['os_family'] based logic ##}
+{% set os_family_map = salt['grains.filter_by']({
+        'RedHat': {
+            "service": {
+                "name": "sysstat",
+                "enabled": False
+            }
+        }
+  }, grain="os_family")
```

Comment:
The reason i left the default in there is because it allows the user, to add as many blocks of such kind as needed. 
> On Apr 6, 2015, at 8:32 PM, Brian Jackson <notifications@github.com> wrote:
> 
> In sysstat/map.jinja <https://github.com/saltstack-formulas/sysstat-formula/pull/9#discussion_r27846033>:
> 
> > -}, merge=salt['pillar.get']('sysstat:lookup')) %}
> > +# -*- coding: utf-8 -*-
> > +# vim: ft=jinja
> > +
> > +{## Start with  defaults from defaults.sls ##}
> > +{% import_yaml 'sysstat/defaults.yml' as default_settings %}
> > +
> > +{## setup variable using grains['os_family'] based logic ##}
> > +{% set os_family_map = salt['grains.filter_by']({
> > +        'RedHat': {
> > +            "service": {
> > +                "name": "sysstat",
> > +                "enabled": False
> > +            }
> > +        }
> > +  }, grain="os_family")
> os_family is the default
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/saltstack-formulas/sysstat-formula/pull/9/files#r27846033>.
>